src/main/java/org/vaadin/harry/spring/views/reportoverview/ReportOverview.java

```diff
@@ -0,0 +1,414 @@
+package org.vaadin.harry.spring.views.reportoverview;
+
+import com.vaadin.flow.component.AbstractField;
+import com.vaadin.flow.component.ClickEvent;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.html.Paragraph;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.polymertemplate.Id;
+import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
+import com.vaadin.flow.component.progressbar.ProgressBar;
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.dom.Element;
+import com.vaadin.flow.router.AfterNavigationEvent;
+import com.vaadin.flow.router.AfterNavigationObserver;
+import com.vaadin.flow.templatemodel.TemplateModel;
+import org.apache.commons.lang3.StringUtils;
+import org.vaadin.bugrap.domain.BugrapRepository;
+import org.vaadin.bugrap.domain.entities.Project;
+import org.vaadin.bugrap.domain.entities.ProjectVersion;
+import org.vaadin.bugrap.domain.entities.Report;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+
+/**
+ * A Designer generated component for the report-overview template.
+ * <p>
+ * Designer will add and remove fields with @Id mappings but
+ * does not overwrite or otherwise change this file.
+ */
+@Tag("report-overview")
+@JsModule("./src/views/report-overview/report-overview.js")
+@CssImport(value = "./styles/views/report-overview/report-overview.css")
+public class ReportOverview extends PolymerTemplate<ReportOverview.ReportOverviewModel> implements AfterNavigationObserver {
+
+    // connect to Bugrap domain service
+    private static BugrapRepository bugrapRepository = new BugrapRepository("/tmp/bugrap;create=true");
+
+    @Id("vaadinComboBox")
+    private static ComboBox<String> projectsComboBox;
```

Comment:
The components shouldn't be static. Each view instance needs to have its own component instances.

Reply:
yes, got it, Tomi. I removed static already and understand why they should not be static.

Reply:
Please also push the changes so I get them to my environment

Reply:
copy sir !